src/components/ProjectDetailPage/ProjectDetailPage.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { Grid, Card, CardTitle, CardBody, FlexItem, Flex } from '@patternfly/react-core';
+import { PageLayout } from './../PageLayout/PageLayout';
+import { useDataContainer } from '../../containers/DataContainer/useDataContainer';
+import { projectService } from '../../services/projectService';
+import { DataContainer } from '../../containers/DataContainer/DataContainer';
+import { ActionHeader } from '../ActionButton/ActionButton';
+
+const ProjectAttribute = ({ attribute, value }: { attribute: string; value: string }) => {
```

Comment:
I think it makes sense to create more general component, the same "issue" needs to be solved on all other places.